<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrientationInput</class>
 <widget class="QDialog" name="OrientationInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oriantation Input</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TITLE</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Cardan angles</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Roll</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="rollDoubleSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Pitch</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="pitchDoubleSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Yaw</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="yawDoubleSpinBox">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Quaternion</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="xDoubleSpinBox">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>0.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="yDoubleSpinBox">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>0.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="zDoubleSpinBox">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>0.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>w</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="wDoubleSpinBox">
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>153</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OrientationInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OrientationInput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pitchDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>OrientationInput</receiver>
   <slot>cardanChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rollDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>OrientationInput</receiver>
   <slot>cardanChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yawDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>OrientationInput</receiver>
   <slot>cardanChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>OrientationInput</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>OrientationInput</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>OrientationInput</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zDoubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>OrientationInput</receiver>
   <slot>quaternionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cardanChanged()</slot>
  <slot>quaternionChanged()</slot>
 </slots>
</ui>
